server/blob/blob-storage-strategy/pom.xml

```diff
@@ -44,6 +48,23 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>blob-memory</artifactId>
```

Comment:
I have a loop cycle dependency here.
- blob-memory <= blob-storage-strategy
- blob-storage-strategy <= blob-memory

should I worry about it?

Reply:
Yes. 

Move the GC algorithm test into another module EG the blob-memory one?